tests/comparisons/Model/testBakeTableWithEnum.php

```diff
@@ -0,0 +1,116 @@
+<?php
+declare(strict_types=1);
+
+namespace Bake\Test\App\Model\Table;
+
+use Bake\Test\App\Model\Enum\BakeArticleStatus;
```

Comment:
I like this convention. It helps ensure that enums are scoped to the entity so that we don't easily get conflicts.